bundles/thunk_bundle.js

```diff
@@ -4293,6 +4293,9 @@ module.exports = (() => {
   // helpers/ensure.ts
   init_buffer_shim();
 
+  // helpers/migration.ts
+  init_buffer_shim();
```

Comment:
this looks like a duplicate of the line above it... hopefully it's idempotent

Reply:
yeah, esbuild injects this whenever a new import is added. it ensures the buffer shim is available. but the generated code does seem stupid because the module has been imported before..